icssc/AntAlmanac
# Improve schedule name dialog

## Summary
To make the schedule name dialog consistent with other dialogs, such as the ones that pop up for saving/loading/importing schedules, the following changes were implemented:
- If the user presses Enter while the dialog is open, the schedule name is submitted.
- When the dialog opens, the cursor is auto-focused onto the text field. The user doesn't need to click on the field to start typing.
- Removed the following feature: if the user wants to add a schedule and clicks on the text field, the default schedule name is erased.

Some other small changes:
- The `handleAdd` and `handleRename` functions were combined into the `submitName` function.
- The `handleClose` function was renamed to `handleCancel`. `handleClose` is a bit misleading since it only runs when the cancel button is pressed.

## Test Plan
- Make sure the user can add a schedule by pressing Enter or clicking the submit button normally.
- Make sure the user can rename a schedule by pressing Enter or clicking the submit button normally.

## Issues

Closes #547
